definitions/npm/body-parser_v1.x.x/flow_v0.17.x-/body-parser_v1.x.x.js

```diff
@@ -1,33 +1,28 @@
-import type { Middleware, $Request, $Response } from 'express';
-
-declare type bodyParser$Options = {
-  inflate?: boolean;
-  limit?: number | string;
-  type?: string | string[] | ((req: $Request) => any);
-  verify?: (req: $Request, res: $Response, buf: Buffer, encoding: string) => void;
-};
-
-declare type bodyParser$OptionsText = bodyParser$Options & {
-  reviver?: (key: string, value: any) => any;
-  strict?: boolean;
-};
-
-declare type bodyParser$OptionsJson = bodyParser$Options & {
-  reviver?: (key: string, value: any) => any;
-  strict?: boolean;
-};
-
-declare type bodyParser$OptionsUrlencoded = bodyParser$Options & {
-  extended?: boolean;
-  parameterLimit?: number;
-};
-
 declare module "body-parser" {
 
-    declare type Options = bodyParser$Options;
-    declare type OptionsText = bodyParser$OptionsText;
-    declare type OptionsJson = bodyParser$OptionsJson;
-    declare type OptionsUrlencoded = bodyParser$OptionsUrlencoded;
+    import type { Middleware, $Request, $Response } from 'express';
```

Comment:
Place the import statement on the first line of the file, outside of the module declaration.

Reply:
Wouldn't this defeat the purpose of this PR?

Reply:
Oh man I totally read that, and forgot it for the review. Either way, specs are failing since it can't find the import, and I can't merge this in until they pass.